examples/data-transfer-pg/main.py

```diff
@@ -3,8 +3,13 @@
 import os
 import time
 import pyodbc
+from faker import Faker
+from typing import List, Tuple
+
 
 CONNECTION_STRING = 'DRIVER={{PostgreSQL Unicode}};SERVER={server};DATABASE={database};UID={username};PWD={password};'
+RECORD_COUNT = 10000
+SQL_INSERT_DATA = 'INSERT INTO users (id, name, city) VALUES ({id}, {name}, {city});'
```

Comment:
Use prepared query as much as possible and bind the parameters instead.
```suggestion
SQL_INSERT_DATA = 'INSERT INTO users (id, name, city) VALUES (?, ?, ?);'
```